xcpmd/configure.ac

```diff
@@ -40,6 +40,8 @@ LT_INIT
 # Check for libraries
 AC_SEARCH_LIBS([dlopen], [dl])
 AC_SEARCH_LIBS([yajl_tree_parse], [yajl])
+AC_SEARCH_LIBS([round], [m])
+AC_SEARCH_LIBS([udev_new], [udev])
```

Comment:
udev could be added in with the Makefile.am change a few commits earlier.

Seems like https://github.com/OpenXT/xctools/pull/74/commits/153e6b66984588495df74a7459e7659fc71232e2 is fixes that could be folded in earlier.

Reply:
As discussed, you're ok with me not splitting this out right? Or do you still want me to move the AC_SEARCH_LIBS to an earlier commit?

Reply:
Unless you want to address it, you don't have to.  Since these are ~6 year old commits which you didn't write, I feel bad you have to clean it up.

In general though, I'd like review comments to be addressed by modifying the original commit and not just tacking additional patches on the end.

Reply:
> Unless you want to address it, you don't have to. Since these are ~6 year old commits which you didn't write, I feel bad you have to clean it up.

I already rolled up most of the cleanup commits that were in our downstream into the larger feature commits, because I wouldn't want all those "fix this, fix that" commits cluttering the history.  This one seemed to have its own relevance so I left it separate.

> In general though, I'd like review comments to be addressed by modifying the original commit and not just tacking additional patches on the end.

Can do.  The whitespace fixes I can break up and put them into the commits that introduced them originally.

Reply:
That all sounds good.  Thanks for moving the whitespace fixups.